pom.xml

```diff
@@ -409,39 +400,91 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<version>1.5.22.RELEASE</version>
 			<scope>test</scope>
-            <exclusions>
-                <exclusion>
-                    <groupId>org.assertj</groupId>
-                    <artifactId>assertj-core</artifactId>
-                </exclusion>
-            </exclusions>
-		</dependency>
-		<dependency>
-             <groupId>org.springframework</groupId>
-             <artifactId>spring-aspects</artifactId>
-             <version>${spring.framework.version}</version>
-         </dependency>
-	 	 <dependency>
+			<exclusions>
+				<exclusion>
+					<groupId>org.assertj</groupId>
+					<artifactId>assertj-core</artifactId>
+				</exclusion>
+			</exclusions>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-aspects</artifactId>
+			<version>${spring.framework.version}</version>
+		</dependency>
+		<dependency>
 			<groupId>org.assertj</groupId>
 			<artifactId>assertj-core</artifactId>
 			<version>3.18.1</version>
 			<scope>test</scope>
-		 </dependency>
+		</dependency>
+		<!-- Feign -->
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-starter-openfeign</artifactId>
+			<version>${spring-cloud-starter-openfeign}</version>
+			<exclusions>
+				<exclusion>
+					<groupId>ch.qos.logback</groupId>
+					<artifactId>logback-classic</artifactId>
+				</exclusion>
+				<exclusion>
+					<groupId>org.slf4j</groupId>
+					<artifactId>slf4j-parent</artifactId>
+				</exclusion>
+				<exclusion>
+					<groupId>log4j</groupId>
+					<artifactId>log4j</artifactId>
+				</exclusion>
+				<exclusion>
+					<groupId>org.slf4j</groupId>
+					<artifactId>slf4j-log4j12</artifactId>
+				</exclusion>
+				<exclusion>
+					<groupId>org.slf4j</groupId>
+					<artifactId>log4j-over-slf4j</artifactId>
+				</exclusion>
```

Comment:
something misterious here, feign seems only depending on slf4j in its [pom.xml](https://github.com/OpenFeign/feign/blob/master/pom.xml)... I will check on my local

Reply:
found out it would be sufficient to comment out `slf4j-log4j12` dependency and leave feign alone without exclusions, could you try?

Reply:
Hi @mwithi, as you suggested I tried to exclude only slf4j-log4j12 dependency and removed the exclusion of other libraries from feign, but I still have an exception when I try to un OH-GUI (_Caused by: java.lang.IllegalStateException: Detected both log4j-over-slf4j.jar AND bound slf4j-log4j12.jar on the class path, preempting StackOverflowError. See also http://www.slf4j.org/codes.html#log4jDelegationLoop for more details._). So that I returned to my version.